release/models/interfaces/openconfig-if-ip.yang

```diff
@@ -1224,6 +1244,54 @@ revision "2023-06-30" {
     }
   }
 
+  grouping urpf-config {
+    description
+        "Grouping configuration coniguration of URPF";
+    leaf enabled {
+      type boolean;
+      default false;
+      description
+        "Turns ON/OFF packet's source address verification against content of
+        FIB. This is one of anti-spoofing protection techniques.";
+    }
+    leaf mode {
+      type urpf-mode;
+      default STRICT;
```

Comment:
Here and throughout, we have an OC bias for new models against specifying defaults, unless this is defined by RFC.  I recommend we remove the default and require the client to specify the node they want.